packages/web-components/fast-foundation/src/toolbar/toolbar.ts

```diff
@@ -198,6 +189,31 @@ export class Toolbar extends FoundationElement {
         return true;
     }
 
+    /**
+     * get all the slotted elements
+     * @internal
+     */
+    protected getAllSlottedItems(): (HTMLElement | Node)[] {
+        return [
+            ...this.start.assignedNodes(),
+            ...this.slottedItems,
+            ...this.end.assignedNodes(),
+        ];
+    }
+
+    /**
+     * Prepare the slotted elements which can be focusable.
+     *
+     * @internal
+     */
+    protected buildFocusableElements(): void {
```

Comment:
`reduceFocusableElements` might be a clearer indication of what's going on with this method.

Reply:
agreed!